src/encoded/static/components/body_map.js

```diff
@@ -237,6 +237,10 @@ const highlightOrgan = (e) => {
     }
 };
 
+// Checks to see if "b" (string) is included in "selectedOrgan" (string or list)
+// "checkClass" is used to set an active class on buttons or images based on whether or not they are a selected organ
+const checkClass = (selectedOrgan, b) => ((typeof selectedOrgan === 'string' && selectedOrgan === b) || (typeof selectedOrgan !== 'string' && selectedOrgan.includes(b)));
```

Comment:
You need not worry about this. Your way is also probably clearer. 

Pardon me if this appears patronizing. It is not my intent.

I assume by "list" you mean "array".  Anyway, a string is an array-like object  (like an array of characters) and you can do array-like operations of it. For example, both of these return true

'xyz'.includes('xyz') // string
['xyz'].includes('xyz') //array

So in theory, you need not typecheck. Pardon me, I missed that in my previous review. On the downside, I have not done this much with using strings as an array other than iterator over a string. So, I do not know the edge cases.

'b' is a good when you pass as argument (what you pass in when a function/method is called). But as parameters (what you use in a function/method definition), it may be unclear

Reply:
Oh yes, this can be simpler. I think I needed `typeof` somewhere else because spread syntax was spreading my arrays (when I combined them) into things like ["h", "e", "a", "r", "t", "lungs"] but here there is no need!

Reply:
I almost feel like this doesn't need to be a method now that it's so short, but I think it's fine in either case.